invenio_records_lom/api.py

```diff
@@ -18,10 +18,10 @@
 from invenio_records.models import RecordMetadata
 from werkzeug.local import LocalProxy
 
-from invenio_records_lom.models import LomMetadata
-
 from .signals import lomrecord_created
 
+# from invenio_records_lom.models import LomMetadata
```

Comment:
why only comment it? why not removing it?

Reply:
There is still a decision to be made, if it makes sense to have a new table in the database rather than using the records table.
 
New commits will add a revised To do : on these commented codes.